code/modules/paperwork/ticketmachine.dm

```diff
@@ -167,9 +167,11 @@
 		user.adjust_fire_stacks(1)
 		user.IgniteMob()
 
+/obj/machinery/ticket_machine/attack_ai(mob/user)//AI cannot interact with ticket_machine
+	return
 /obj/item/ticket_machine_ticket
 	name = "Ticket"
-	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper�. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
+	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper®. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
```

Comment:
What changed here?

Reply:
Encoding got changed, I changed it back but I had removed the corrupted symbol, so I put it back.

Reply:
I'd prefer this not show up at all.